### ISSUE TITLE
1:00 Min timeout timer - only run once and cant be interrupted

### ISSUE THREAD

**Issue by SMD2023** [msaltnet/handball-stopwatch#2]
On smartwatch it is only possible to run the timer once. 
Unlike the source code (see issue "1:00 Min Timeout timer - Time is 10 instead of 60 in source code") and the test in the browser, the timer on the clock runs 60 seconds and only ONCE. 

**WHY WE NEED THE TIMEOUT TIMER**
There are several reasons to stop the time during the game. These include, for example: 
- Player is injured
- The playing field needs to be cleaned
- A coach takes a timeout
Therefore, it should be possible to interrupt the main timer at any time. 
To keep it simple we DO NOT decide the reason for the timeout.
BUT
The timeout of the coach lasts 60 seconds. The countdown/timeouttimer and alarm after 60 seconds is only to remind the referee that 60 seconds are over. 
Because
In any case, the referee manually resumes the maintimer and continues the game. 

Because we do not decide the reason it must be possible to start the maintimer everytime.

**Comment by SMD2023** [comment]
For interruption I used variable
    interruptTimoutTimer = true;   
in an earlier version. 

Start MainTimer will change interruptTimeoutTimer variable.
While the if loop is running and adding -1 every second it is necessary to check of the interruptTimoutTimer has changed. 
If it has changed (because start timer change the value) the timeouttimer stops and set to 1:00.

**Comment by SMD2023** [comment]
see new issue including an example code
